<launch>
    <!-- <node pkg="topic_tools" type="mux" name="cmd_vel_mux" output="screen">
        <param name="output" value="/ypspur_ros/cmd_vel" />
        <param name="topics" value="['/cmd_vel_source_normal', '/cmd_vel_source_avoidance']" />
    </node> -->

    <!-- <node pkg="topic_tools" type="mux" name="mux_cmd_vel" args="ypspur_ros/cmd_vel /cmd_vel_source_normal /cmd_vel_source_avoidance mux:=mux_cmd_vel"/>]
     -->

    <!-- mycode for mux -->
    <node name="cmdvel_mux" pkg="openai_test" type="cmd_vel_mux" output="screen" />

</launch>
